### DISCUSSION TITLE
novo com flutter

### DISCUSSION THREAD

**Discussion by Mejinka** [Flutterando/forum#74] [Perguntas e Respostas]
boas, sou novo em flutter e queria alguma ajuda pra tentar fazer a ligação com um banco de dados, não entendi algumas coisas, outras ficaram meio confusas, se alguem puder ajudar, agradeço muito

**Comment by eduardoflorence** [comment]
@Mejinka tenta mostrar seu código para vermos como tentou fazer. Qual é o banco de dados?

**Comment by FredericoWP** [comment]
> boas, sou novo em flutter e queria alguma ajuda pra tentar fazer a ligação com um banco de dados, não entendi algumas coisas, outras ficaram meio confusas, se alguém puder ajudar, agradeço muito

Boa tarde @Mejinka !
Conseguiu resolver essa situação ?

Falando sobre banco de dados é muito importante que nos informe qual é o banco que esta usando e se trata de um banco local ou remoto.

Aguardamos sua interação para melhor te ajudar!
